README.md

```diff
@@ -1,4 +1,4 @@
-[ ![Download](https://api.bintray.com/packages/twilio/releases/video-android/images/download.svg) ](https://bintray.com/twilio/releases/video-android/1.3.5)
+[ ![Download](https://api.bintray.com/packages/twilio/releases/video-android/images/download.svg) ](https://bintray.com/twilio/releases/video-android/1.3.6)
```

Comment:
We should remove the download badge because it always points to latest so it's either wrong for `master` or `2.0` depending on which release is latest. The docs badge is handy though.